apps/mobile/src/locales/en/messages.po

```diff
@@ -0,0 +1,46 @@
+msgid ""
+msgstr ""
+"POT-Creation-Date: 2024-06-05 16:21+0400\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: @lingui/cli\n"
+"Language: en\n"
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Plural-Forms: \n"
+
+#: src/components/welcome-screen/index.tsx:327
+msgid "<0><1>Bitcoin for</1></0><2><3>the rest of us</3></2>"
+msgstr "<0><1>Bitcoin for</1></0><2><3>the rest of us</3></2>"
+
+#: src/components/welcome-screen/index.tsx:252
+msgid "Done"
+msgstr "Done"
+
+#: src/components/welcome-screen/index.tsx:286
+msgid "Email address"
+msgstr "Email address"
+
+#: src/components/welcome-screen/index.tsx:240
+msgid "Follow us"
+msgstr "Follow us"
+
+#: src/components/welcome-screen/index.tsx:341
+msgid "Leather is the only wallet you need to tap into the multilayered Bitcoin economy"
+msgstr "Leather is the only wallet you need to tap into the multilayered Bitcoin economy"
+
+#: src/components/welcome-screen/index.tsx:261
+msgid "Stay in the loop and be the first one to hear about new developments"
+msgstr "Stay in the loop and be the first one to hear about new developments"
+
+#: src/components/welcome-screen/index.tsx:296
+msgid "Submit"
+msgstr "Submit"
+
+#: src/components/welcome-screen/index.tsx:226
+msgid "Thanks for subscribing! We'll notify you when Leather's mobile app launches. In the meantime, please <0>check out our browser extension.</0>"
+msgstr "Thanks for subscribing! We'll notify you when Leather's mobile app launches. In the meantime, please <0>check out our browser extension.</0>"
```

Comment:
There's no need to commit this if we're generating the file right? 

Would be much nicer to incorporate generation into build flow, as it works with Panda, rather than committing generated files. This can cause big repo-bloat and conflict issues.

Reply:
What happens if we human author updates some text, and another then re-generates the file?